bread/views/browse.py

```diff
@@ -67,23 +67,28 @@ class BrowseView(BreadView, LoginRequiredMixin, PermissionListMixin, ListView):
     orderingurlparameter: str = "ordering"
     objectids_urlparameter: str = "_selected"  # see bread/static/js/main.js:submitbulkaction and bread/layout/components/datatable.py
     bulkaction_urlparameter: str = "_bulkaction"
-    items_per_page_options: Tuple[int] = None
+    items_per_page_options: Optional[Tuple[int]] = None
```

Comment:
And this should be ```Optional[Iterable[int]]``` too I think. It is used to declare which options for the "N-items per page" dropdown will show.